OracleWebLogic/samples/12212-msiserver/Dockerfile.addapp

```diff
@@ -0,0 +1,50 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This Dockerfile extends the Oracle WebLogic MSI server image by adding app
+# bits to the image and recording it in domain configuration
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Run the following command for the sample application included in this directory. The name and source identify
+# the name of the applciation to be used and the application source. simple_file identified the simple file name
+# that will be used to copy the source file on the image. It can be the same as the last element of application
+# source
+#      $ sudo docker build -t 12212-summercamps-msiserver -f Dockerfile.addapp --build-arg name=summercamps  --build-arg source=apps/summercamps.ear --build-arg simple_filename=summercamps.ear .
+#
+
+# Pull base image
+# ---------------
+FROM 12212-msiserver:latest
+
+# Maintainer
+# ----------
+MAINTAINER Aseem Bajaj <aseem.bajaj@oracle.com>
+
+# Arguments
+# ---------
+ARG name
```

Comment:
Yes, that's right Avi. These are arguments that specify what application to deploy and from where. So there can't be any defaults for these.